src/EntityParamConverter.php

```diff
@@ -27,7 +27,7 @@ public function supports(string $class): bool
     /**
      * @inheritDoc
      */
-    public function convertTo(string $value, string $class)
+    public function convertTo(string $value, string $class, $default = null)
```

Comment:
@Ali1 I would like to avoid introducing this change right now. As I have mentioned, 1) If the request param value is missing and the corresponding method attribute is optional, then the default value will be applied by PHP (when invoking the controller action). Otherwise, If the request param is not missing then it should be either converted to a new value or passed as it is, without involving the defined default value.

Reply:
👍